Scripts/Source/UD_AbadonWeapon_AME.psc

```diff
@@ -18,7 +18,7 @@ Function OnEffectStart(Actor akTarget, Actor akCaster)
         if UDmain.TraceAllowed()
             UDmain.Log(GetActorName(akTarget) + " hit by abadon weapon. Locking device")
         endif
-        UDmain.UDRRM.LockRandomRestrain(loc_actor,false,0x00000080) ;lock hand restrain
+        UDmain.UDRRM.LockRandomRestrain(loc_actor,false,0x00000010) ;lock hand restraint
```

Comment:
Either update the UDRRM or undo the changes (see comment in UDRRM)

Reply:
Of course.

Reply:
Done.
Tested, working.